appyx-components/spotlight/common/src/commonMain/kotlin/com/bumble/appyx/components/spotlight/gestures/Gestures.kt

```diff
@@ -0,0 +1,58 @@
+package com.bumble.appyx.components.spotlight.gestures
+
+import androidx.compose.foundation.gestures.Orientation
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.unit.Density
+import com.bumble.appyx.components.spotlight.SpotlightModel.State
+import com.bumble.appyx.components.spotlight.operation.Next
+import com.bumble.appyx.components.spotlight.operation.Previous
+import com.bumble.appyx.interactions.core.ui.context.TransitionBounds
+import com.bumble.appyx.interactions.core.ui.gesture.Drag
+import com.bumble.appyx.interactions.core.ui.gesture.Gesture
+import com.bumble.appyx.interactions.core.ui.gesture.GestureFactory
+import com.bumble.appyx.interactions.core.ui.gesture.dragHorizontalDirection
+import com.bumble.appyx.interactions.core.ui.gesture.dragVerticalDirection
+
+class Gestures<InteractionTarget>(
```

Comment:
Not necessarily beneficial to think this will cover all spotlight gesture cases to make it an extracted, central one. This factory is specific to swiping elements moving a full width/height distance. For any other representation this will not work and be misleading. I'd rather just move it back to `SpotlightSlider`, and import it from there for any other case where it's needed.